Docs/CommentCommands.md

```diff
@@ -27,9 +27,9 @@ Make the generated type-safe accessor `public`:
 container.register(MyType.self, factory: { /*...*/ })
 ```
 
-#### Getter Named
+#### Getter Alias
 
-* `getter-named("myCustomName")`: Generate a named accessor function with a custom name where the default name is not appropriate. Provide the custom name as an argument to the command.
+* `alias("myCustomName")`: Generate an additional accessor function with a custom alias when the default name is not ideal. Provide the custom name as an argument to the command.
```

Comment:
It doesn't look like this PR makes the change to always keep the original accessor, but I think it will be more consistent if we follow this comment.

Reply:
Nevermind, I just noticed the other PR to do this.